packages/fa-icon-chooser/src/utils/utils.ts

```diff
@@ -28,12 +28,20 @@ export function buildDefaultIconsSearchResult(familyStyles: object): object {
 
   const icons = get(defaultIconsSearchResult, 'data.search', []);
 
-  for (const i of icons) {
+  // Filter out bluesky and web-awesome icons if version is less than 6.5.2
```

Comment:
default icons were updated to include blue sky and web-awesome.
But those icons were not available prior to 6.5.2.
The code below will filter out those icons for version prior to 6.5.2.
Without the filter a console error appears about not being able to find those icons.